src/models/Option.js

```diff
@@ -28,17 +29,24 @@ export default class Option {
     if (event.status !== EVENT_STATUS.ARBITRATION) {
       const totalBalance = _.sum(event.roundBets);
       this.percent = totalBalance === 0 ? totalBalance : _.round((this.amount / totalBalance) * 100);
+      this.userPercent = this.amount === 0 ? this.amount : _.round((event.userRoundBets[i] / this.amount) * this.percent);
     } else {
       this.isPrevResult = event.currentResultIndex === i;
+      this.value = this.isPrevResult ? event.previousRoundBets[i] : this.amount;
```

Comment:
why is there an `amount` and `value`? the naming seems so similar it's confusing to someone as to what each purpose is for.

Reply:
seems like the amount is only for temp use.

Reply:
hm can we get rid of it then?